README.md

```diff
@@ -1,2 +1,23 @@
-# Angular2-Scaffold
-Angular 2 Scaffold by ngParty team
+# Angular2 Scaffold
+
+## Install
+
+```bash
+$ npm install
+```
+
+## Start Development
+
+```bash
+$ npm run web:dev
+```
+
+open http://localhost:8080
```

Comment:
hmm this is not windows friendly, hmmm or maybe it works in gitbash, I dunno, hyd!